executor/builder.go

```diff
@@ -185,3 +191,22 @@ func (b *executorBuilder) buildLimit(v *plan.Limit) Executor {
 	}
 	return e
 }
+
+func (b *executorBuilder) buildPrepare(v *plan.Prepare) Executor {
+	return &PrepareExec{
+		Ctx:     b.ctx,
+		IS:      b.is,
+		Name:    v.Name,
+		SQLText: v.SQLText,
+	}
+}
+
+func (b *executorBuilder) buildExecute(v *plan.Execute) Executor {
```

Comment:
buildExecute is only for stmt executor? if only, I prefer adding stmt for better understanding.